src/main/java/org/opensearch/security/auth/BackendRegistry.java

```diff
@@ -345,7 +345,7 @@ && isBlocked(((InetSocketAddress) request.getHttpChannel().getRemoteAddress()).g
                 return false;
             }
 
-            final String tenant = Utils.coalesce(request.header("securitytenant"), request.header("security_tenant"));
+            final String tenant = Optional.ofNullable(request.header("securitytenant")).orElse(request.header("security_tenant"));
```

Comment:
Move to a function? Seems easy to forget to update an instance of this header in other changes